Remotion/Web/Development.WebTesting/Accessibility/AccessibilityAnalyzer.cs

```diff
@@ -220,13 +220,17 @@ private bool AxeIsInjected ()
     /// <summary>
     /// Injects Axe into every iframe manually because axe-core cannot do it itself.
     /// </summary>
+    /// <param name="source">Source of axe.min.js to inject.</param>
+    /// <param name="maximumTimeToWaitForFrame">Maximum wait time after which an iframe must be visible such that Axe can be injected. Default is 3000 ms.</param>
     /// <remarks>
     /// Source: https://www.deque.com/blog/writing-automated-tests-accessibility/
     /// </remarks>
-    private void InjectIntoIFrames (string source)
+    private void InjectIntoIFrames (string source, TimeSpan? maximumTimeToWaitForFrame = null)
```

Comment:
In the AccessibilityAnalyzer, we need to allow injecting the value into the public members. It's just that it's OK to define the default value in here. Although, the longer I think about it, the less I like it though, that we define the default in the private method. Maybe it would be best to make a static field for the default and apply it at the call sites when the public method is passed a null-value. 

=> each public API gets a nullable timeout parameter. When the parameter is null, we null-propagate the static-field as fallback.

Reply:
Okay, it seems I misunderstood your previous comment a bit. I hope it's okay now.

Fyi, I have implemented your comment only tentatively in a new commit and will move the change to the second commit ("bug fix") if you okay it. (I think you wouldn't get a nice diff with only the new change if I had rebased it just now)